ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+    "   1. [configure command-line tasks](https://pipelines.lsst.io/v/w-2018-12/modules/lsst.pipe.base/command-line-task-config-howto.html) for your science case\n",
+    "   2. TODO\n",
+    "   \n",
+    "Other techniques that are demonstrated, but not empasized, in this notebook are\n",
+    "   1. Use the `butler` to fetch data\n",
+    "   2. Visualize data with the LSST Stack\n",
+    "   3. TODO\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "\n",
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import eups.setupcmd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I: Interacting with data. Introduction to the Butler\n",
+    "https://pipelines.lsst.io/getting-started/data-setup.html\n",
+    "\n",
+    "Part I runs the following command-line tasks\n",
+    "```\n",
+    "eups list lsst_distrib\n",
+    "setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "echo \"lsst.obs.hsc.HscMapper\" > /home/jmyles/DATA/_mapper\n",
+    "ingestImages.py /home/jmyles/DATA /home/jmyles/repositories/ci_hsc/raw/*.fits --mode=link\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/CALIB/ /home/jmyles/DATA/CALIB\n",
+    "installTransmissionCurves.py /home/jmyles/DATA\n",
+    "mkdir -p /home/jmyles/DATA/ref_cats\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/ps1_pv3_3pi_20170110 /home/jmyles/DATA/ref_cats/ps1_pv3_3pi_20170110\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!eups list lsst_distrib\n",
+    "datarepo = \"/home/jmyles/repositories/ci_hsc/\"\n",
+    "datadir = \"/home/jmyles/DATA/\"\n",
+    "os.system(\"mkdir -p {}\".format(datadir))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "\n",
+    "setup = eups.setupcmd.EupsSetup([\"-j\",\"-r\", datarepo])\n",
+    "status = setup.run()\n",
+    "print('setup exited with status {}'.format(status))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Butler needs a *mapper* file \"to find and organize data in a format specific to each camera.\" We write this file to the data repository so that any instantiated Butler object knows which mapper to use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(datadir + \"_mapper\", \"w\") as f:\n",
```

Comment:
I'd set the mapper file name in a variable here, and then call `open` separately - that way we'll know what file to look for in teh folder.